### ISSUE TITLE
Listen to unix domain socket option

### ISSUE THREAD

**Issue by Cadynum** [photoview/photoview#1018]
A low priority, nice to have, change would be to allow Photoview to listen to unix sockets instead of IP sockets.
When running behind a reverse proxy, this allows the application to be completely isolated from the network, and access to be controlled by file-system permissions. There are probably minor performance benefits too.

I have no go-experience, but perhaps this is a 5-liner code change.

**Comment by googollee** [comment]
I agree with this feature, but it's not an easy change. The Unix socket doesn't work with dev mode now.

I can try to implement when I have time.

**Comment by kkovaletp** [comment]
@Cadynum thanks for the feature request. I just saw this new feature in a new release of another open-source Python project on Github today)
This would be a good security enhancement and probably some performance improvement (but I believe that this is not the bottleneck for the product now)
